cimr/processor/lift.py

```diff
@@ -0,0 +1,84 @@
+"""Update genomic positions to hg38 reference.
+Uses pyliftover 0.4 currently, which only allows
+lifting over point coordinates and not indels, etc.
```

Comment:
What is `indels`? (Just want to confirm that it's not a typo.)

Reply:
insertion-deletion of >1 nucleotide bases. it's a genomics jargon.